engines/stark/ui/menu/fmvmenu.cpp

```diff
@@ -0,0 +1,167 @@
+/* ResidualVM - A 3D game interpreter
+ *
+ * ResidualVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the AUTHORS
+ * file distributed with this source distribution.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ *
+ */
+
+#include "engines/stark/ui/menu/fmvmenu.h"
+
+#include "engines/stark/services/services.h"
+#include "engines/stark/services/userinterface.h"
+#include "engines/stark/services/diary.h"
+
+namespace Stark {
+
+FMVMenuScreen::FMVMenuScreen(Gfx::Driver *gfx, Cursor *cursor) :
+		StaticLocationScreen(gfx, cursor, "DiaryFMV", Screen::kScreenFMVMenu) {
```

Comment:
Please make sure all the fields are initialized (even to zero) in the constructor (Here it's mostly to avoid warnings from the static code analysis tools, but it's a good practice anyway).